### PR TITLE
ODQA readme

### PR DESCRIPTION
**Pull request by my-master**

### CODE REVIEW

File: deeppavlov/skills/odqa/README.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+* **train** - parameters for vectorizer fitting
+    * **_validate_best_**- is ingnored, any value
+    * **_test_best_** - is ignored, any value
+    * **_batch_size_** - how many Wikipedia articles should return the dataset iterator in a single batch
+
+### ODQA
+
+Default ODQA config for **English** language is `deeppavlov/configs/odqa/en_odqa_infer_prod.json`
+
+Default ODQA config for **Russian** language is `deeppavlov/configs/odqa/en_odqa_infer_prod.json`
```
Comment (yoptar) [inline deeppavlov/skills/odqa/README.md]: ru?

File: deeppavlov/skills/odqa/README.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+### enwiki.db
+
+**enwiki.db** SQLite database consists of **5159530 Wikipedia articles**
+and is built by the following steps:
+1. Download a Wikipedia dump file. We took the latest [enwiki](https://dumps.wikimedia.org/enwiki/20180201)
+ (from 2018-02-11)
+2. Unpack and extract the articles with [WikiExtractor](https://github.com/attardi/wikiextractor)
+ (with `--json`, `--no-templates`, `--filter_disambig_pages` options)
+3. Build a database.
```
Comment (yoptar) [inline deeppavlov/skills/odqa/README.md]: Looks like this block should either say less (that .db was built from this dump and that's it) or explain how to build a database in more detail.

File: deeppavlov/skills/odqa/README.md (hunk 1 above)
Comment (seliverstov) [reply deeppavlov/skills/odqa/README.md]: Link to Russian config is wrong

File: deeppavlov/skills/odqa/README.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+Answer:
+> Luke Skywalker
+
+## Languages
+
+There are pretrained ODQA models for **English** and **Russian** languages in DeepPavlov.
+
+## Models
+
+The architecture of ODQA skill is modular and consists of two models, a ranker and a rader. The ranker is based on
```
Comment (seliverstov) [inline deeppavlov/skills/odqa/README.md]: `rader` -> reader

File: deeppavlov/skills/odqa/README.md (hunk 2 above)
Comment (seliverstov) [reply deeppavlov/skills/odqa/README.md]: Agree.

File: deeppavlov/skills/odqa/README.md (hunk 1 above)
Comment (my-master) [reply deeppavlov/skills/odqa/README.md]: done

File: deeppavlov/skills/odqa/README.md (hunk 3 above)
Comment (my-master) [reply deeppavlov/skills/odqa/README.md]: done

**Pull request merged**